spec/ruby/language/regexp_spec.rb

```diff
@@ -147,4 +147,24 @@
     pattern.should_not =~ 'fooF'
     pattern.should_not =~ 'T'
   end
+
+  ALL_TERMINATORS = (("!".."/").to_a + (":".."@").to_a + ("[".."`").to_a + ("{".."~").to_a)
```

Comment:
This defines a constant on Object which is not acceptable.
Please use local variables instead and move in the example as possible (only `ESCAPABLE_TERMINATORS` is shared).

Reply:
FWIW,
```ruby
[*("!".."/"), *(":".."@"), *("[".."`"), *("{".."~")]
```
is another way to write this.